main/src/test/java/org/mobilitydata/gtfsvalidator/validator/ShapeIncreasingDistanceValidatorTest.java

```diff
@@ -34,7 +36,7 @@ public static GtfsShape createShapePoint(
       double shapePtLat,
       double shapePtLon,
       int shapePtSequence,
-      double shapeDistTraveled) {
+      Double shapeDistTraveled) {
```

Comment:
Sorry for coming in late here, but is there a reason this was changed from `double` to `Double`?

Reply:
This was done in order to support testing agains null value.